recipes/asyncinit/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "asyncinit" %}
+{% set version = "0.2.4" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/asyncinit-{{ version }}.tar.gz
+  sha256: 4acaf614f4d6f78babd278bbacf21bd9f7756efc17cd5d7e1bae31ff770b3cfa
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - python >=3.5
+
+test:
+  imports:
+    - asyncinit
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/kchmck/pyasyncinit
+  summary: Class decorator to enable async __init__
+  doc_url: https://pythonhosted.org/asyncinit/
+  license: MIT
+  license_file: 
```

Comment:
This should probably be changed to `license_file: License` and then you'll need to download the file from the repo (https://github.com/kchmck/pyasyncinit/blob/master/License) and include it the package folder.

Reply:
Let me know if you need any help with that! :)

Reply:
Ah, didn't realize I need to download it - thought it would find it! Thanks so much for the tip, I'll add that :)

Reply:
Pleasure! Yeah it depends on whether it's included in the original package or not. (https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually)